docs/output.md

```diff
@@ -37,6 +56,344 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
+
+It is used in nf-core/taxprofiler for complexity filtering using different algorithms. This means that it will remove reads with low sequence diversity (e.g. mono- or dinucleotide repeats).
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `bbduk/`
+  - `<sample_id>.bbduk.log`: log file containing filtering statistics
+  - `<sample_id>.fastq.gz`: resulting FASTQ file without low-complexity reads
+
+</details>
+
+By default nf-core/taxprofiler will only provide the `.log` file if BBDuk is selected as the complexity filtering tool. You will only find the complexity filtered reads in your results directory if you provide ` --save_complexityfiltered_reads` .
+
+Note that the FASTQ file(s) may _not_ always be the 'final' reads that go into taxprofiling, if you also run other steps such as host removal, run merging etc..
+
+### PRINSEQ++
+
+[PRINSEQ++](https://github.com/Adrian-Cantu/PRINSEQ-plus-plus) is a C++ implementation of the [prinseq-lite.pl](https://prinseq.sourceforge.net/) program. It can be used to filter, reformat or trim genomic and metagenomic sequence data.
+
+It is used in nf-core/taxprofiler for complexity filtering using different algorithms. This means that it will remove reads with low sequence diversity (e.g. mono- or dinucleotide repeats).
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `prinseqplusplus/`
+  - `<sample_id>.log`: log file containing number of reads. Row IDs correspond to: `min_len, max_len, min_gc, max_gc, min_qual_score, min_qual_mean, ns_max_n, noiupac, derep, lc_entropy, lc_dust, trim_tail_left, trim_tail_right, trim_qual_left, trim_qual_right, trim_left, trim_right`
+  - `<sample_id>_good_out.fastq.gz`: resulting FASTQ file without low-complexity reads
+
+</details>
+
+By default nf-core/taxprofiler will only provide the `.log` file if PRINSEQ++ is selected as the complexity filtering tool. You will only find the complexity filtered FASTQ files in your results directory if you supply ` --save_complexityfiltered_reads` .
+
+Note that the FASTQ file(s) may _not_ always be the 'final' reads that go into taxprofiling, if you also run other steps such as host removal, run merging etc..
+
+### Filtlong
+
+[Filtlong](https://github.com/rrwick/Filtlong) is a quality filtering tool for long reads. It can take a set of small reads and produce a smaller, better subset.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `filtlong`
+  - `<sample_id>_filtered.fastq.gz`: Quality or short read data filtered file
+  - `<sample_id>_filtered.log`: log file containing summary statistics
+
+</details>
+
+We do **not** recommend using Filtlong if you are performing filtering of low quality reads with ONT's basecaller Guppy.
```

Comment:
```suggestion
You can use the filtered FASTQ for other downstream analyses to reduce repeated preprocessing of files.

We do **not** recommend using Filtlong if you are performing filtering of low quality reads with ONT's basecaller Guppy.
```
See suggesiton, but thinking about it - does the filtlong/guppy warning need to go here? Would'nt you want to know this _before_ running the pipeline, not after?

Reply:
You are right!

Reply:
I think we have adressed this warning in the usage section as well.